packages/flutter_tools/lib/src/base/common.dart

```diff
@@ -17,7 +17,7 @@ Never throwToolExit(String message, { int? exitCode }) {
 class ToolExit implements Exception {
   ToolExit(this.message, { this.exitCode });
 
-  final String message;
+  final String? message;
```

Comment:
`throwToolExit(null)` in a bunch of places.

Reply:
oops